Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -494,6 +537,11 @@ internal static void Log(string toPrint)
             UnityEngine.Debug.Log(toPrint);
         }
 
+        /// <summary>
+        /// Checks for new Peer2Peer messages for Session management, as well as updating the OnlineSessionState of ActiveSessions.
+        /// This Manager is not a MonoBehaviour, so this function should be run by the program every update.
```

Comment:
The comment is correct, and this is certainly outside the scope of the task of adding comments to existing code - but it brings up a point that might otherwise not have been noticed: is it possible that `EOSSessionsManager` _should_ be a `MonoBehaviour` (which would obviate the need to make this comment)?